fjord/base/l10n.py

```diff
@@ -11,3 +16,19 @@ class Translation(object):
 
     import jingo
     jingo.env.install_gettext_translations(Translation)
+
+
+def strip_whitespace(message):
+    return re.compile(r'\s+', re.UNICODE).sub(' ', message).strip()
+
+
+class MozInternationalizationExtension(InternationalizationExtension):
+    """
+    We override jinja2's _parse_block() to collapse whitespace so we can have
+    linebreaks wherever we want.
+    """
+
+    def _parse_block(self, parser, allow_pluralize):
+        parse_block = InternationalizationExtension._parse_block
+        ref, buffer = parse_block(self, parser, allow_pluralize)
+        return ref, strip_whitespace(buffer)
```

Comment:
That's does look like the right way... I'll change it and make sure stuff still works